tidepods/make_pfs.py

```diff
@@ -0,0 +1,146 @@
+import os
+import subprocess
+import sys
+
+from . import generate_pts as gp
+
+
+def generatepfs(infile, date, mikepath, tempdir):
+    """Generates a pfs file using DHI.PFS
+
+    Parameters
+    ----------
+    infile : str
+        Path to AOI polygon.
+
+    date : datetime.datetime
+        Image acqusition date and time
+
+    mikepath : str
+        Path to MIKE installation directory
+
+    tempdir : str
+        Path to temporary directory
+
+    Returns
+    -------
+    temppfs :  str
+        Path to the generate pfs file within the tempdir
+
+    """
+
+    temppfs = os.path.join(tempdir, 'temp.pfs')
+
+    sdkpath = os.path.join(mikepath, r'MIKE SDK\bin')
+    constituents_path = os.path.join(mikepath,
+                                     r'MIKE Zero\Application Data\Tide_Constituents\global_tide'
+                                     '_constituents_0.125deg.dfs2')
+    prepack_path = os.path.join(mikepath,
+                                r'MIKE Zero\Application Data\Tide_Constituents\prepack.dat')
+
+    import clr
+
+    clr.AddReference('System')
+
+    import System
+
+    try:
+        sys.path.insert(0, sdkpath)
+
+    except System.IO.FileNotFoundException as exception:
```

Comment:
Better just check whether the path exists with `os.path.isdir` or so beforehand

Reply:
Ok done